mage.go

```diff
@@ -3,9 +3,85 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+    "strings"
 	"github.com/magefile/mage/sh"
+    "github.com/magefile/mage/mg"
 )
 
+// emoji related constants
+const emojiList = "https://raw.githubusercontent.com/CodeFreezr/emojo/master/db/v5/emoji-v5.json"
+const emojiFileName = "emojiList.go"
+
+// json parsing structure
+type emoji struct {
+	Num         int    `json:"No"`
+	Emoji       string `json:"Emoji"`
+	Category    string `json:"Category"`
+	SubCategory string `json:"SubCategory"`
+	Unicode     string `json:"Unicode"`
+	Name        string `json:"Name"`
+	Tags        string `json:"Tags"`
+	Shortcode   string `json:"Shortcode"`
+}
+
+// This func downloaded and parses the emojis from online into a slice of all shortnames
+// to be used as a lookup for tab completion for emojis
+// this way the pull from GitHub only has to be done at build time.
+func createEmojiSlice() ([]string, error) {
+	result, err := http.Get(emojiList)
+	if err != nil {
+		return nil, err
+	}
+	defer result.Body.Close()
+
+	emojiList, err := ioutil.ReadAll(result.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	var emojis []emoji
+	if err := json.Unmarshal(emojiList, &emojis); err != nil {
+		return nil, err
+	}
+
+	var emojiSlice []string
+
+	for _, emj := range emojis {
+		if len(emj.Shortcode) == 0 || strings.Contains(emj.Shortcode, "_tone") {
+			// dont add them
+            continue
+		}
+		emojiSlice = append(emojiSlice, emj.Shortcode)
+	}
+	return emojiSlice, nil
+}
+
+// Build kbtui with emoji lookup support
```

Comment:
This comment is what shows when you run `mage -l` but running this option doesn't actually build kbtui, it just fetches the emoji list so it can be included when mage builds. I think this should be made clear here.